src/main/java/com/chessmates/controller/EntityController.groovy

```diff
@@ -22,12 +20,12 @@ class EntityController {
      * Return a list of all team players
      */
     @GetMapping(value = 'players')
-    List<Player> getPlayers() { entityService.getPlayers() }
+    List getPlayers() { entityService.getPlayers() }
```

Comment:
Ok, I see, thi sis not the way in Groovy. Perhaps maybe we should  be clear that this is actually getPlayersJson() method?